Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"ru\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\r\n    <link rel=\"stylesheet\" href=\"styles/index.css\">\r\n</head>\r\n<body class=\"page\">\r\n\r\n    <header class=\"header\">\r\n        <img src=\"images/Vector.svg\" alt=\"Логотип MESTO RUSSIA\" class=\"header__logo\">\r\n    </header>\r\n\r\n    <main>\r\n\r\n\r\n\r\n    </main>\r\n\r\n    <footer class=\"footer\">\r\n        <p class=\"footer__copyright\">&copy;&nbsp;2022 Mesto Russia</p>\r\n    </footer>\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision 93c145fba0d59fed3ff53308b60f13aac1dcede8)
+++ b/index.html	(date 1643911719831)
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles/index.css">
+    <title>MESTO RUSSIA</title>
 </head>
 <body class="page">
 
